README.md

```diff
@@ -140,7 +140,16 @@ If you are using JRuby then you need to download latest [Oracle JDBC driver](htt
 TESTS
 -----
 
-* Create Oracle database schema for test purposes. Review `spec/spec_helper.rb` to see default schema/user names and database names (use environment variables to override defaults)
+Review `spec/spec_helper.rb` to see default schema/user names and database names (use environment variables to override defaults)
+
+##### Prepare database
+
+* With local Vagrant based Oracle XE database
```

Comment:
The vagrant is usable only for specific Operating systems.
Can you specify what are the supported OS's?